src/lib/components/PostPreview/PostPreviewCard.tsx

```diff
@@ -1,129 +1,135 @@
 import React from 'react';
-import { useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import { ThemeProvider } from '@material-ui/core/styles';
 import Card from '@material-ui/core/Card';
 import Box from '@material-ui/core/Box';
 import CardContent from '@material-ui/core/CardContent';
 import CardMedia from '@material-ui/core/CardMedia';
 import Typography from '@material-ui/core/Typography';
 import { IPost } from '../../types';
 import { useStyles } from '../../styles/PostPreviewCard.styles';
-import PostDirectionLink from '../PostDirectionLink';
+import { MAIN_THEME } from '../../theme/theme';
 
 export interface IPostPreviewCardProps {
   data: IPost;
 }
-
 const PostPreviewCard: React.FC<IPostPreviewCardProps> = (props) => {
   const { data } = props;
-  const history = useHistory();
-
+  let expertLink = ``;
+  const postLink = `/posts/${data.id}`;
   let authorFullName = '';
+
   if (data.author?.firstName && data.author?.lastName) {
     authorFullName = `${data.author?.firstName} ${data.author?.lastName}`;
+    expertLink = `/experts/${data.author.id}`;
```

Comment:
technically this doesn't depend on if check above, so it's correct not to put here

Reply:
modified utilities/types.ts, interface IPost { author: IExpert}, so author is not already optional field. Therefore mentioned checks are no more required